.github/workflows/conformance-gateway-api.yaml

```diff
@@ -337,6 +339,12 @@ jobs:
             --junit-property github_job_step="Run tests (${{ join(matrix.*, ', ') }})" \
             --test 'allow-all-except-world,encryption,packet-drops'
 
+      - name: Features tested
+        uses: ./.github/actions/feature-status
+        with:
+          title: "Summary of all features tested"
+          json-filename: "${{ env.job_name }} (${{ join(matrix.*, ', ') }})"
```

Comment:
@Artyop I'm not following, why would it be a problem?

Reply:
I guess just an old habit of me about an ancient tale of portability between systems but since we're never going to use them anywhere else it's not a problem